<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mEqWidget</class>
 <widget class="QWidget" name="mEqWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mEqWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Highpass freq:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Lowpass freq:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="lowpass_doubleSpinBox">
         <property name="minimum">
          <double>20.000000000000000</double>
         </property>
         <property name="maximum">
          <double>20000.000000000000000</double>
         </property>
         <property name="value">
          <double>20000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="highpass_doubleSpinBox">
         <property name="minimum">
          <double>20.000000000000000</double>
         </property>
         <property name="maximum">
          <double>20000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bandFilterLayout"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>EQ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
